public/js/components/Editor.css

```diff
@@ -42,10 +41,6 @@
   background-color: var(--theme-body-background);
 }
 
-.CodeMirror-linenumber {
-  font-size: 11px;
-}
```

Comment:
I see, thanks for the info.  I removed it because size 11px makes the gutter font smaller than the size of text in the main panel to the right of the gutter, hence the height difference.  I'll try adding line-height instead.